opennms-base-assembly/src/main/filtered/etc/poller-configuration.xml

```diff
@@ -284,10 +294,16 @@
          <parameter key="rrd-repository" value="${install.share.dir}/rrd/response"/>
          <parameter key="ds-name" value="minion-rpc"/>
       </service>
-      <downtime begin="0" end="300000" interval="30000"/><!-- 30s, 0, 5m -->
-      <downtime begin="300000" end="43200000" interval="300000"/><!-- 5m, 5m, 12h -->
-      <downtime begin="43200000" end="432000000" interval="600000"/><!-- 10m, 12h, 5d -->
-      <downtime begin="432000000" interval="3600000"/><!-- 1h, 5d -->
+      <service name="JMX-Minion" interval="30000" user-defined="false" status="on">
+         <parameter key="port" value="18980"/>
+         <parameter key="retry" value="2"/>
+         <parameter key="timeout" value="3000"/>
+         <parameter key="rrd-repository" value="${install.share.dir}/rrd/response"/>
+         <parameter key="ds-name" value="jmx-minion"/>
+         <parameter key="use-foreign-id-as-system-id" value="true"/>
+      </service>
+      <!-- Query every 30secs always -->
+      <downtime begin="0" interval="30000"/><!-- 30s -->
```

Comment:
Do we need to set a downtime model if we just want to stick with the same interval?

Reply:
`downtime` is not an optional field in `poller-configuration`